README.md

```diff
@@ -28,6 +28,7 @@ Project statistics fetched from [Ohloh](https://www.ohloh.net).
 |Cables|[Cables communication](http://dee.su/cables) implements secure and anonymous communication using email-like addresses, pioneered in Liberté Linux. Cables communication is Liberté's pivotal component for enabling anyone to communicate safely and covertly in hostile environments.|-|-|-|-|-|
 |Camlistore|[Camlistore](http://camlistore.org/) is your personal storage system for life. It is an acronym for "Content-Addressable Multi-Layer Indexed Storage" and could be described as "Like git for all content in your life"|Go|5 K|344 K|82 |5 year(s)|
 |ClearSkies|[ClearSkies](https://github.com/jewel/clearskies) is a peer-to-peer file sync program.  It is inspired by BitTorrent Sync, but has an open and fully-documented protocol.|C|849 |241 K|18 |2 year(s)|
+|Cloudron|[Cloudron](http://cloudron.io/) is a platform to run apps on your server. It includes 1-Click app install, automatic backups, updates, Single Sign-On, DNS setup, SSL provisioning and a secure firewall. |-|-|-|-|-|
```

Comment:
Could you please elaborate a bit more on how is Cloudron more decentralized than any other VPS or container hosting panel?